### PR TITLE
Add log_files plugin to auto-generation

### PR DESCRIPTION
**Pull request by julianoes**

### CODE REVIEW

File: protos/log_files/log_files.proto [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    repeated Entry entries = 2; // List of entries
+}
+
+message SubscribeDownloadLogFileRequest {
+    uint32 id = 1; // ID of the log file to download
+    string path = 2; // Path of where to download log file to.
+}
+message DownloadLogFileResponse {
+    LogFileResult log_file_result = 1;
+    float progress = 2; // Progress if result is progress
```
Comment (JonasVautherin) [inline protos/log_files/log_files.proto:34]: This assumes that we will never return anything other than this `float`. Fine for me, but just to make sure :-)

Comment (julianoes) [reply protos/log_files/log_files.proto:34]: I would think so, unless download speed :thinking: ...

Comment (julianoes) [reply protos/log_files/log_files.proto:34]: I'm changing it to a `ProgressData`.

**Comment by julianoes** [comment]
Thx.

**Pull request merged**